src/v/archival/ntp_archiver_service.cc

```diff
@@ -44,21 +44,26 @@ namespace archival {
 
 ntp_archiver::ntp_archiver(
```

Comment:
Would it be possible to describe briefly why implemented approach is better than the previous one in commit message ?